plone/namedfile/scaling.py

```diff
@@ -326,4 +326,8 @@ def tag(self,
             direction='thumbnail',
             **kwargs):
         scale = self.scale(fieldname, scale, height, width, direction)
+        if height is not None:
+            kwargs['height'] = height
+        if width is not None:
+            kwargs['width'] = width
```

Comment:
This doesn't look right to me. The generated scale fits within the requested dimensions, but does not necessarily have those exact dimensions if the image has a different aspect ratio than the requested dimensions.